Update HandlerMappingIntrospector Usage in CORS support

evgeniycheban opened the pull request:
Closes gh-16501

<!--
For Security Vulnerabilities, please use https://pivotal.io/security#reporting
-->

<!--
Before creating new features, we recommend creating an issue to discuss the feature. This ensures that everyone is on the same page before extensive work is done.

Thanks for contributing to Spring Security. Please provide a brief description of your pull-request and reference any related issue numbers (prefix references with gh-).
-->

bodograumann committed:
Avoid UriComponentsBuilder.fromUri

Closes gh-15852

kse-music committed:
Annotation parameter scan finds first-level conflicts

Closes PR-16312

jzheaux committed:
Align Method Traversal with MergedAnnotations

Closes gh-16751

franticticktick committed:
Add Support BadCredentialsException to OneTimeTokenAuthenticationProvider

Closes gh-16494

Signed-off-by: Max Batischev <mblancer@mail.ru>

franticticktick committed:
Add Support ServerFormPostRedirectStrategy

Closes gh-16542

Signed-off-by: Max Batischev <mblancer@mail.ru>

franticticktick committed:
Add setRedirectStrategy to OidcClientInitiatedServerLogoutSuccessHandler

Closes gh-16556

Signed-off-by: Max Batischev <mblancer@mail.ru>

matthewgreene committed:
Fix ordering for security filter configuration

Signed-off-by: Matthew Greene <mgreene57005@gmail.com>
Signed-off-by: mgreene57005 <mgreene57005@gmail.com>

big-cir committed:
Make mapToUser and mapToGrantedAuthority protected in JdbcUserDetailsManager
- Closes gh-16540

Signed-off-by: dae won <eodnjs01477@gmail.com>

rwinch committed:
PortResolver.NO_OP

Closes gh-16666

yhao3 committed:
Ensure ID Token is updated after refresh token

Signed-off-by: Hao <kyrieeeee2@gmail.com>

big-cir committed:
Add userDetailsMapper as a class member

Signed-off-by: dae won <eodnjs01477@gmail.com>

big-cir committed:
Refactor mapToUser method visibility to private

Signed-off-by: dae won <eodnjs01477@gmail.com>

AB-xdev committed:
Correct Closing Tag

Closes gh-16600

Signed-off-by: AB <a.bierler@xdev-software.de>

big-cir committed:
Add unit tests for setUserDetailsMapper method

Signed-off-by: dae won <eodnjs01477@gmail.com>

big-cir committed:
Add grantedAuthorityMapper as a class member
- Add unit tests for setGrantedAuthorityMapper method

Signed-off-by: dae won <eodnjs01477@gmail.com>

franticticktick committed:
Add Support SingleResultAuthorizationManager

Closes gh-16590

Signed-off-by: Max Batischev <mblancer@mail.ru>

Juha-1 committed:
Update logout.adoc

typos

Signed-off-by: Juha-1 <52188855+Juha-1@users.noreply.github.com>

yelm-212 committed:
Update Http403ForbiddenEntryPoint to Use HttpStatus

Closes gh-16615

Signed-off-by: yelm-212 <21yrshin@naver.com>

github-actions[bot] committed:
Release 6.3.7

github-actions[bot] committed:
Next development version

pat-mccusker committed:
Add fourth oauth grant type to javadoc

Signed-off-by: Pat McCusker <patmccusker14@gmail.com>

pat-mccusker committed:
Use possessive pronoun rather contraction

Signed-off-by: Pat McCusker <patmccusker14@gmail.com>

franticticktick committed:
Fix JdbcUserCredentialRepository Save

Closes gh-16620

Signed-off-by: Max Batischev <mblancer@mail.ru>

sjohnr committed:
Merge branch '6.3.x' into 6.4.x

sjohnr committed:
Merge branch '6.4.x'

sjohnr committed:
Polish gh-16551

franticticktick committed:
Make DefaultOneTimeToken Serializable

Closes gh-16617

Signed-off-by: Max Batischev <mblancer@mail.ru>

jzheaux committed:
Polish Exception Message

This error message provides more context around which credentials failed.

Issue gh-16494

jzheaux committed:
JwtDecoders Supports Hostnames with Underscores

In the process of verifying gh-15852, another issue with URI was discovered.
This commit adds tests to the uri-computing methods and changes them to use
UriComponents instead of URI.

Issue gh-15852

jzheaux committed:
ClientRegistrations supports hostnames with underscores

Issue gh-15852

dependabot[bot] committed:
Bump org.seleniumhq.selenium:selenium-java from 4.28.1 to 4.29.0

Bumps [org.seleniumhq.selenium:selenium-java](https://github.com/SeleniumHQ/selenium) from 4.28.1 to 4.29.0.
- [Release notes](https://github.com/SeleniumHQ/selenium/releases)
- [Commits](https://github.com/SeleniumHQ/selenium/commits/selenium-4.29.0)

---
updated-dependencies:
- dependency-name: org.seleniumhq.selenium:selenium-java
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

Kuba15 committed:
Fix @PostResult example in method-security

Replace @PreFilter with @Postfilter in example

Signed-off-by: Olivier <Kuba15@users.noreply.github.com>

jzheaux committed:
Add PathPatterRequestMatcher

Closes gh-16429
Clsoes gh-16430

jzheaux committed:
Add Opt-in PathPattern Strategy

Closes gh-16573

jzheaux commented on config/src/main/java/org/springframework/security/config/annotation/web/configurers/CorsConfigurer.java:
```diff
@@ -110,12 +111,12 @@ private static CorsFilter getMvcCorsFilter(ApplicationContext context) {
-			return new CorsFilter(mappingIntrospector);
+			PreFlightRequestHandler mappingIntrospector = context.getBean(HANDLER_MAPPING_INTROSPECTOR_BEAN_NAME,
+					PreFlightRequestHandler.class);
+			return new CorsFilter((CorsConfigurationSource) mappingIntrospector);
```
Please do a safe cast here, and then throw an exception if it isn't of the right type.

evgeniycheban replied:
Wouldn't it be better to look up for `CorsConfigurationSource`?

jzheaux reviewed (changes requested):
Thanks, @evgeniycheban! I left a comment inline. Also, `gh-16429` has been merged, so would you please open a new PR in Spring Security?

ngocnhan-tran1996 committed:
Fix typo

Signed-off-by: Tran Ngoc Nhan <ngocnhan.tran1996@gmail.com>

dependabot[bot] committed:
Bump @springio/antora-extensions from 1.14.2 to 1.14.4 in /docs

Bumps [@springio/antora-extensions](https://github.com/spring-io/antora-extensions) from 1.14.2 to 1.14.4.
- [Changelog](https://github.com/spring-io/antora-extensions/blob/main/CHANGELOG.adoc)
- [Commits](https://github.com/spring-io/antora-extensions/compare/v1.14.2...v1.14.4)

---
updated-dependencies:
- dependency-name: "@springio/antora-extensions"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump @springio/antora-extensions from 1.14.2 to 1.14.4 in /docs

Bumps [@springio/antora-extensions](https://github.com/spring-io/antora-extensions) from 1.14.2 to 1.14.4.
- [Changelog](https://github.com/spring-io/antora-extensions/blob/main/CHANGELOG.adoc)
- [Commits](https://github.com/spring-io/antora-extensions/compare/v1.14.2...v1.14.4)

---
updated-dependencies:
- dependency-name: "@springio/antora-extensions"
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.3.x' into 6.4.x

github-actions[bot] committed:
Merge branch '6.4.x'

dependabot[bot] committed:
Bump org.htmlunit:htmlunit from 4.9.0 to 4.10.0

Bumps [org.htmlunit:htmlunit](https://github.com/HtmlUnit/htmlunit) from 4.9.0 to 4.10.0.
- [Release notes](https://github.com/HtmlUnit/htmlunit/releases)
- [Commits](https://github.com/HtmlUnit/htmlunit/compare/4.9.0...4.10.0)

---
updated-dependencies:
- dependency-name: org.htmlunit:htmlunit
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.hibernate.orm:hibernate-core from 6.6.8.Final to 6.6.9.Final

Bumps [org.hibernate.orm:hibernate-core](https://github.com/hibernate/hibernate-orm) from 6.6.8.Final to 6.6.9.Final.
- [Release notes](https://github.com/hibernate/hibernate-orm/releases)
- [Changelog](https://github.com/hibernate/hibernate-orm/blob/6.6.9/changelog.txt)
- [Commits](https://github.com/hibernate/hibernate-orm/compare/6.6.8...6.6.9)

---
updated-dependencies:
- dependency-name: org.hibernate.orm:hibernate-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.seleniumhq.selenium:htmlunit3-driver from 4.28.0 to 4.29.0

Bumps [org.seleniumhq.selenium:htmlunit3-driver](https://github.com/SeleniumHQ/htmlunit-driver) from 4.28.0 to 4.29.0.
- [Release notes](https://github.com/SeleniumHQ/htmlunit-driver/releases)
- [Commits](https://github.com/SeleniumHQ/htmlunit-driver/compare/4.28.0...4.29.0)

---
updated-dependencies:
- dependency-name: org.seleniumhq.selenium:htmlunit3-driver
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump io.freefair.gradle:aspectj-plugin from 8.12.1 to 8.12.2

Bumps [io.freefair.gradle:aspectj-plugin](https://github.com/freefair/gradle-plugins) from 8.12.1 to 8.12.2.
- [Release notes](https://github.com/freefair/gradle-plugins/releases)
- [Commits](https://github.com/freefair/gradle-plugins/compare/8.12.1...8.12.2)

---
updated-dependencies:
- dependency-name: io.freefair.gradle:aspectj-plugin
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.hibernate.orm:hibernate-core from 6.6.8.Final to 6.6.9.Final

Bumps [org.hibernate.orm:hibernate-core](https://github.com/hibernate/hibernate-orm) from 6.6.8.Final to 6.6.9.Final.
- [Release notes](https://github.com/hibernate/hibernate-orm/releases)
- [Changelog](https://github.com/hibernate/hibernate-orm/blob/6.6.9/changelog.txt)
- [Commits](https://github.com/hibernate/hibernate-orm/compare/6.6.8...6.6.9)

---
updated-dependencies:
- dependency-name: org.hibernate.orm:hibernate-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.4.x'

jzheaux committed:
Merge branch '6.3.x' into 6.4.x

jzheaux committed:
Merge branch '6.4.x'

jzheaux committed:
Merge branch '6.4.x'

jzheaux committed:
Update Copyright

Issue gh-16600

jzheaux committed:
Merge branch '6.4.x'

jzheaux committed:
Fix JavaDoc Typos

Commit:
Support Custom RequestMatchers for WebAuthn

Closes gh-16517

Signed-off-by: topiam <support@topiam.cn>

jzheaux committed:
Add Type Validation

Closes gh-16672

dependabot[bot] committed:
Bump ch.qos.logback:logback-classic from 1.5.16 to 1.5.17

Bumps [ch.qos.logback:logback-classic](https://github.com/qos-ch/logback) from 1.5.16 to 1.5.17.
- [Release notes](https://github.com/qos-ch/logback/releases)
- [Commits](https://github.com/qos-ch/logback/compare/v_1.5.16...v_1.5.17)

---
updated-dependencies:
- dependency-name: ch.qos.logback:logback-classic
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.slf4j:slf4j-api from 2.0.16 to 2.0.17

Bumps org.slf4j:slf4j-api from 2.0.16 to 2.0.17.

---
updated-dependencies:
- dependency-name: org.slf4j:slf4j-api
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.4.x'

dependabot[bot] committed:
Bump ch.qos.logback:logback-classic from 1.5.16 to 1.5.17

Bumps [ch.qos.logback:logback-classic](https://github.com/qos-ch/logback) from 1.5.16 to 1.5.17.
- [Release notes](https://github.com/qos-ch/logback/releases)
- [Commits](https://github.com/qos-ch/logback/compare/v_1.5.16...v_1.5.17)

---
updated-dependencies:
- dependency-name: ch.qos.logback:logback-classic
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.slf4j:slf4j-api from 2.0.16 to 2.0.17

Bumps org.slf4j:slf4j-api from 2.0.16 to 2.0.17.

---
updated-dependencies:
- dependency-name: org.slf4j:slf4j-api
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.3.x' into 6.4.x

github-actions[bot] committed:
Merge branch '6.4.x'

github-actions[bot] committed:
Merge branch '6.3.x' into 6.4.x

github-actions[bot] committed:
Merge branch '6.4.x'

dependabot[bot] committed:
Bump ch.qos.logback:logback-classic from 1.5.16 to 1.5.17

Bumps [ch.qos.logback:logback-classic](https://github.com/qos-ch/logback) from 1.5.16 to 1.5.17.
- [Release notes](https://github.com/qos-ch/logback/releases)
- [Commits](https://github.com/qos-ch/logback/compare/v_1.5.16...v_1.5.17)

---
updated-dependencies:
- dependency-name: ch.qos.logback:logback-classic
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

jzheaux committed:
Use Thread-Safe Map

Issue gh-15906

jzheaux committed:
Merge branch '6.4.x'

evgeniycheban committed:
Update HandlerMappingIntrospector Usage in CORS support

Closes gh-16501

Signed-off-by: Evgeniy Cheban <mister.cheban@gmail.com>

evgeniycheban commented:
@jzheaux I opened a PR to Spring Security repository

rwinch committed:
Add migration-7/web.adoc to nav

Closes gh-16661

rwinch committed:
Fix migration source type

Previously the migration-7/web.adoc source type was kotlin for the xml
source. This commit fixes that.

Closes gh-16662

rwinch committed:
Ad section summary toc

Closes gh-16663

rwinch committed:
Use PortResolver Beans by Default

Closes gh-16664

sjohnr committed:
Add support for automatic context-propagation with Micrometer

Closes gh-16665

rwinch committed:
Deprecate PortResolver

Closes gh-15972

dependabot[bot] committed:
Bump org.jfrog.buildinfo:build-info-extractor-gradle

Bumps [org.jfrog.buildinfo:build-info-extractor-gradle](https://github.com/jfrog/build-info) from 4.33.23 to 4.33.24.
- [Release notes](https://github.com/jfrog/build-info/releases)
- [Changelog](https://github.com/jfrog/build-info/blob/master/RELEASE.md)
- [Commits](https://github.com/jfrog/build-info/compare/build-info-gradle-extractor-4.33.23...build-info-gradle-extractor-4.33.24)

---
updated-dependencies:
- dependency-name: org.jfrog.buildinfo:build-info-extractor-gradle
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.jfrog.buildinfo:build-info-extractor-gradle

Bumps [org.jfrog.buildinfo:build-info-extractor-gradle](https://github.com/jfrog/build-info) from 4.33.23 to 4.33.24.
- [Release notes](https://github.com/jfrog/build-info/releases)
- [Changelog](https://github.com/jfrog/build-info/blob/master/RELEASE.md)
- [Commits](https://github.com/jfrog/build-info/compare/build-info-gradle-extractor-4.33.23...build-info-gradle-extractor-4.33.24)

---
updated-dependencies:
- dependency-name: org.jfrog.buildinfo:build-info-extractor-gradle
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.4.x'

github-actions[bot] committed:
Merge branch '6.3.x' into 6.4.x

github-actions[bot] committed:
Merge branch '6.4.x'

dependabot[bot] committed:
Bump org.jfrog.buildinfo:build-info-extractor-gradle

Bumps [org.jfrog.buildinfo:build-info-extractor-gradle](https://github.com/jfrog/build-info) from 4.33.23 to 4.34.1.
- [Release notes](https://github.com/jfrog/build-info/releases)
- [Changelog](https://github.com/jfrog/build-info/blob/master/RELEASE.md)
- [Commits](https://github.com/jfrog/build-info/compare/build-info-gradle-extractor-4.33.23...build-info-gradle-extractor-4.34.1)

---
updated-dependencies:
- dependency-name: org.jfrog.buildinfo:build-info-extractor-gradle
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

jzheaux committed:
Add RFC 9068 Support

Closes gh-13185

jzheaux committed:
Polish NimbusJwtDecoder

sjohnr committed:
Polish gh-16589

jzheaux committed:
Fix Tests

Issue gh-16517

dependabot[bot] committed:
Bump io.mockk:mockk from 1.13.16 to 1.13.17

Bumps [io.mockk:mockk](https://github.com/mockk/mockk) from 1.13.16 to 1.13.17.
- [Release notes](https://github.com/mockk/mockk/releases)
- [Commits](https://github.com/mockk/mockk/commits)

---
updated-dependencies:
- dependency-name: io.mockk:mockk
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump io.mockk:mockk from 1.13.16 to 1.13.17

Bumps [io.mockk:mockk](https://github.com/mockk/mockk) from 1.13.16 to 1.13.17.
- [Release notes](https://github.com/mockk/mockk/releases)
- [Commits](https://github.com/mockk/mockk/commits)

---
updated-dependencies:
- dependency-name: io.mockk:mockk
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump io.mockk:mockk from 1.13.16 to 1.13.17

Bumps [io.mockk:mockk](https://github.com/mockk/mockk) from 1.13.16 to 1.13.17.
- [Release notes](https://github.com/mockk/mockk/releases)
- [Commits](https://github.com/mockk/mockk/commits)

---
updated-dependencies:
- dependency-name: io.mockk:mockk
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.3.x' into 6.4.x

github-actions[bot] committed:
Merge branch '6.4.x'

github-actions[bot] committed:
Merge branch '6.4.x'

jzheaux committed:
Add HttpsRedirectFilter

Closes gh-16678

jzheaux committed:
Add redirectToHttps DSL Configurer

Closes gh-16679

jzheaux committed:
Deprecate ChannelSecurityConfigurer

Closes gh-16680

vpavic committed:
Introduce JwtAudienceValidator

Signed-off-by: Vedran Pavic <vedran@vedranpavic.com>

jgrandja committed:
Enforce BCrypt password length

jzheaux committed:
Deprecate ChannelDecisionManager

Closes gh-16681

dependabot[bot] committed:
Bump com.fasterxml.jackson:jackson-bom from 2.18.2 to 2.18.3

Bumps [com.fasterxml.jackson:jackson-bom](https://github.com/FasterXML/jackson-bom) from 2.18.2 to 2.18.3.
- [Commits](https://github.com/FasterXML/jackson-bom/compare/jackson-bom-2.18.2...jackson-bom-2.18.3)

---
updated-dependencies:
- dependency-name: com.fasterxml.jackson:jackson-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump com.webauthn4j:webauthn4j-core

Bumps [com.webauthn4j:webauthn4j-core](https://github.com/webauthn4j/webauthn4j) from 0.28.5.RELEASE to 0.28.6.RELEASE.
- [Release notes](https://github.com/webauthn4j/webauthn4j/releases)
- [Changelog](https://github.com/webauthn4j/webauthn4j/blob/master/github-release-notes-generator.yml)
- [Commits](https://github.com/webauthn4j/webauthn4j/compare/0.28.5.RELEASE...0.28.6.RELEASE)

---
updated-dependencies:
- dependency-name: com.webauthn4j:webauthn4j-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump com.webauthn4j:webauthn4j-core

Bumps [com.webauthn4j:webauthn4j-core](https://github.com/webauthn4j/webauthn4j) from 0.28.5.RELEASE to 0.28.6.RELEASE.
- [Release notes](https://github.com/webauthn4j/webauthn4j/releases)
- [Changelog](https://github.com/webauthn4j/webauthn4j/blob/master/github-release-notes-generator.yml)
- [Commits](https://github.com/webauthn4j/webauthn4j/compare/0.28.5.RELEASE...0.28.6.RELEASE)

---
updated-dependencies:
- dependency-name: com.webauthn4j:webauthn4j-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump com.fasterxml.jackson:jackson-bom from 2.18.2 to 2.18.3

Bumps [com.fasterxml.jackson:jackson-bom](https://github.com/FasterXML/jackson-bom) from 2.18.2 to 2.18.3.
- [Commits](https://github.com/FasterXML/jackson-bom/compare/jackson-bom-2.18.2...jackson-bom-2.18.3)

---
updated-dependencies:
- dependency-name: com.fasterxml.jackson:jackson-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.4.x'

github-actions[bot] committed:
Merge branch '6.4.x'

dependabot[bot] committed:
Bump io.freefair.gradle:aspectj-plugin from 8.12.2 to 8.12.2.1

Bumps [io.freefair.gradle:aspectj-plugin](https://github.com/freefair/gradle-plugins) from 8.12.2 to 8.12.2.1.
- [Release notes](https://github.com/freefair/gradle-plugins/releases)
- [Commits](https://github.com/freefair/gradle-plugins/compare/8.12.2...8.12.2.1)

---
updated-dependencies:
- dependency-name: io.freefair.gradle:aspectj-plugin
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.mockito:mockito-bom from 5.15.2 to 5.16.0

Bumps [org.mockito:mockito-bom](https://github.com/mockito/mockito) from 5.15.2 to 5.16.0.
- [Release notes](https://github.com/mockito/mockito/releases)
- [Commits](https://github.com/mockito/mockito/compare/v5.15.2...v5.16.0)

---
updated-dependencies:
- dependency-name: org.mockito:mockito-bom
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

franticticktick committed:
Polish AbstractAuthenticationTargetUrlRequestHandler

PR gh-16557

Signed-off-by: Max Batischev <mblancer@mail.ru>

jzheaux committed:
SpEL Propagates Authorization Exceptions

Closes gh-16697

jzheaux committed:
Merge branch '6.3.x' into 6.4.x

jzheaux committed:
Merge branch '6.4.x'

dependabot[bot] committed:
Bump org.slf4j:slf4j-api from 2.0.16 to 2.0.17

Bumps org.slf4j:slf4j-api from 2.0.16 to 2.0.17.

---
updated-dependencies:
- dependency-name: org.slf4j:slf4j-api
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.hibernate.orm:hibernate-core from 6.6.9.Final to 6.6.10.Final

Bumps [org.hibernate.orm:hibernate-core](https://github.com/hibernate/hibernate-orm) from 6.6.9.Final to 6.6.10.Final.
- [Release notes](https://github.com/hibernate/hibernate-orm/releases)
- [Changelog](https://github.com/hibernate/hibernate-orm/blob/6.6.10/changelog.txt)
- [Commits](https://github.com/hibernate/hibernate-orm/compare/6.6.9...6.6.10)

---
updated-dependencies:
- dependency-name: org.hibernate.orm:hibernate-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.hibernate.orm:hibernate-core from 6.6.9.Final to 6.6.10.Final

Bumps [org.hibernate.orm:hibernate-core](https://github.com/hibernate/hibernate-orm) from 6.6.9.Final to 6.6.10.Final.
- [Release notes](https://github.com/hibernate/hibernate-orm/releases)
- [Changelog](https://github.com/hibernate/hibernate-orm/blob/6.6.10/changelog.txt)
- [Commits](https://github.com/hibernate/hibernate-orm/compare/6.6.9...6.6.10)

---
updated-dependencies:
- dependency-name: org.hibernate.orm:hibernate-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.4.x'

dependabot[bot] committed:
Bump io.micrometer:micrometer-observation from 1.14.4 to 1.14.5

Bumps [io.micrometer:micrometer-observation](https://github.com/micrometer-metrics/micrometer) from 1.14.4 to 1.14.5.
- [Release notes](https://github.com/micrometer-metrics/micrometer/releases)
- [Commits](https://github.com/micrometer-metrics/micrometer/compare/v1.14.4...v1.14.5)

---
updated-dependencies:
- dependency-name: io.micrometer:micrometer-observation
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.4.x'

dependabot[bot] committed:
Bump io.micrometer:micrometer-observation from 1.14.4 to 1.14.5

Bumps [io.micrometer:micrometer-observation](https://github.com/micrometer-metrics/micrometer) from 1.14.4 to 1.14.5.
- [Release notes](https://github.com/micrometer-metrics/micrometer/releases)
- [Commits](https://github.com/micrometer-metrics/micrometer/compare/v1.14.4...v1.14.5)

---
updated-dependencies:
- dependency-name: io.micrometer:micrometer-observation
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump io.projectreactor:reactor-bom from 2023.0.15 to 2023.0.16

Bumps [io.projectreactor:reactor-bom](https://github.com/reactor/reactor) from 2023.0.15 to 2023.0.16.
- [Release notes](https://github.com/reactor/reactor/releases)
- [Commits](https://github.com/reactor/reactor/compare/2023.0.15...2023.0.16)

---
updated-dependencies:
- dependency-name: io.projectreactor:reactor-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump io.freefair.gradle:aspectj-plugin from 8.12.2.1 to 8.13

Bumps [io.freefair.gradle:aspectj-plugin](https://github.com/freefair/gradle-plugins) from 8.12.2.1 to 8.13.
- [Release notes](https://github.com/freefair/gradle-plugins/releases)
- [Commits](https://github.com/freefair/gradle-plugins/compare/8.12.2.1...8.13)

---
updated-dependencies:
- dependency-name: io.freefair.gradle:aspectj-plugin
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump io.projectreactor:reactor-bom from 2023.0.15 to 2023.0.16

Bumps [io.projectreactor:reactor-bom](https://github.com/reactor/reactor) from 2023.0.15 to 2023.0.16.
- [Release notes](https://github.com/reactor/reactor/releases)
- [Commits](https://github.com/reactor/reactor/compare/2023.0.15...2023.0.16)

---
updated-dependencies:
- dependency-name: io.projectreactor:reactor-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.4.x'

jzheaux committed:
Remove s101 From Builds

Issue gh-16752

jzheaux committed:
Fix Compilation Error

Issue gh-16697

jzheaux committed:
Merge branch '6.3.x' into 6.4.x

jzheaux committed:
Merge branch '6.4.x'

jzheaux committed:
Merge branch '6.4.x'

jgrandja committed:
Merge branch '6.3.x' into 6.4.x

jgrandja committed:
Merge branch '6.4.x'

rwinch committed:
Disable Flaky WebAuthnWebDriverTests

Closes gh-16753

rwinch committed:
Merge branch '6.4.x'

- Disable Flaky WebAuthnWebDriverTests

Closes gh-16754

rwinch committed:
Add testRuntimeOnly junit-platform-launcher

Closes gh-16755

rwinch committed:
Merge branch '6.3.x' into 6.4.x

- adb303e Add testRuntimeOnly junit-platform-launcher

Closes gh-16756

rwinch committed:
Merge branch '6.4.x'

- adb303e Add testRuntimeOnly junit-platform-launcher

Closes gh-16757

dependabot[bot] committed:
Bump io.projectreactor:reactor-bom from 2023.0.15 to 2023.0.16

Bumps [io.projectreactor:reactor-bom](https://github.com/reactor/reactor) from 2023.0.15 to 2023.0.16.
- [Release notes](https://github.com/reactor/reactor/releases)
- [Commits](https://github.com/reactor/reactor/compare/2023.0.15...2023.0.16)

---
updated-dependencies:
- dependency-name: io.projectreactor:reactor-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.hibernate.orm:hibernate-core from 6.6.10.Final to 6.6.11.Final

Bumps [org.hibernate.orm:hibernate-core](https://github.com/hibernate/hibernate-orm) from 6.6.10.Final to 6.6.11.Final.
- [Release notes](https://github.com/hibernate/hibernate-orm/releases)
- [Changelog](https://github.com/hibernate/hibernate-orm/blob/6.6.11/changelog.txt)
- [Commits](https://github.com/hibernate/hibernate-orm/compare/6.6.10...6.6.11)

---
updated-dependencies:
- dependency-name: org.hibernate.orm:hibernate-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.mockito:mockito-bom from 5.16.0 to 5.16.1

Bumps [org.mockito:mockito-bom](https://github.com/mockito/mockito) from 5.16.0 to 5.16.1.
- [Release notes](https://github.com/mockito/mockito/releases)
- [Commits](https://github.com/mockito/mockito/compare/v5.16.0...v5.16.1)

---
updated-dependencies:
- dependency-name: org.mockito:mockito-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.springframework.data:spring-data-bom from 2024.0.9 to 2024.0.10

Bumps [org.springframework.data:spring-data-bom](https://github.com/spring-projects/spring-data-bom) from 2024.0.9 to 2024.0.10.
- [Release notes](https://github.com/spring-projects/spring-data-bom/releases)
- [Commits](https://github.com/spring-projects/spring-data-bom/compare/2024.0.9...2024.0.10)

---
updated-dependencies:
- dependency-name: org.springframework.data:spring-data-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.hibernate.orm:hibernate-core from 6.6.10.Final to 6.6.11.Final

Bumps [org.hibernate.orm:hibernate-core](https://github.com/hibernate/hibernate-orm) from 6.6.10.Final to 6.6.11.Final.
- [Release notes](https://github.com/hibernate/hibernate-orm/releases)
- [Changelog](https://github.com/hibernate/hibernate-orm/blob/6.6.11/changelog.txt)
- [Commits](https://github.com/hibernate/hibernate-orm/compare/6.6.10...6.6.11)

---
updated-dependencies:
- dependency-name: org.hibernate.orm:hibernate-core
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.springframework.data:spring-data-bom from 2024.1.3 to 2024.1.4

Bumps [org.springframework.data:spring-data-bom](https://github.com/spring-projects/spring-data-bom) from 2024.1.3 to 2024.1.4.
- [Release notes](https://github.com/spring-projects/spring-data-bom/releases)
- [Commits](https://github.com/spring-projects/spring-data-bom/compare/2024.1.3...2024.1.4)

---
updated-dependencies:
- dependency-name: org.springframework.data:spring-data-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.4.x'

github-actions[bot] committed:
Merge branch '6.3.x' into 6.4.x

dependabot[bot] committed:
Bump org.springframework:spring-framework-bom from 6.1.17 to 6.1.18

Bumps [org.springframework:spring-framework-bom](https://github.com/spring-projects/spring-framework) from 6.1.17 to 6.1.18.
- [Release notes](https://github.com/spring-projects/spring-framework/releases)
- [Commits](https://github.com/spring-projects/spring-framework/compare/v6.1.17...v6.1.18)

---
updated-dependencies:
- dependency-name: org.springframework:spring-framework-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.springframework:spring-framework-bom from 6.2.3 to 6.2.4

Bumps [org.springframework:spring-framework-bom](https://github.com/spring-projects/spring-framework) from 6.2.3 to 6.2.4.
- [Release notes](https://github.com/spring-projects/spring-framework/releases)
- [Commits](https://github.com/spring-projects/spring-framework/compare/v6.2.3...v6.2.4)

---
updated-dependencies:
- dependency-name: org.springframework:spring-framework-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.4.x'

dependabot[bot] committed:
Bump org.springframework:spring-framework-bom from 6.2.3 to 6.2.4

Bumps [org.springframework:spring-framework-bom](https://github.com/spring-projects/spring-framework) from 6.2.3 to 6.2.4.
- [Release notes](https://github.com/spring-projects/spring-framework/releases)
- [Commits](https://github.com/spring-projects/spring-framework/compare/v6.2.3...v6.2.4)

---
updated-dependencies:
- dependency-name: org.springframework:spring-framework-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Merge branch '6.3.x' into 6.4.x

github-actions[bot] committed:
Merge branch '6.4.x'

github-actions[bot] committed:
Merge branch '6.4.x'

rwinch committed:
opensamlFiveTest.extendsFrom testRuntimeOnly

Issue gh-16756

rwinch committed:
Merge branch '6.4.x'

- 04f530b opensamlFiveTest.extendsFrom testRuntimeOnly

Issue gh-16757

dependabot[bot] committed:
Bump org.junit:junit-bom from 5.11.4 to 5.12.1

Bumps [org.junit:junit-bom](https://github.com/junit-team/junit5) from 5.11.4 to 5.12.1.
- [Release notes](https://github.com/junit-team/junit5/releases)
- [Commits](https://github.com/junit-team/junit5/compare/r5.11.4...r5.12.1)

---
updated-dependencies:
- dependency-name: org.junit:junit-bom
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

github-actions[bot] committed:
Release 6.5.0-M3

github-actions[bot] committed:
Release 6.4.4

github-actions[bot] committed:
Next development version

github-actions[bot] committed:
Next development version

dependabot[bot] committed:
Bump org-eclipse-jetty from 11.0.24 to 11.0.25

Bumps `org-eclipse-jetty` from 11.0.24 to 11.0.25.

Updates `org.eclipse.jetty:jetty-server` from 11.0.24 to 11.0.25

Updates `org.eclipse.jetty:jetty-servlet` from 11.0.24 to 11.0.25

---
updated-dependencies:
- dependency-name: org.eclipse.jetty:jetty-server
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: org.eclipse.jetty:jetty-servlet
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org.springframework.data:spring-data-bom from 2024.1.3 to 2024.1.4

Bumps [org.springframework.data:spring-data-bom](https://github.com/spring-projects/spring-data-bom) from 2024.1.3 to 2024.1.4.
- [Release notes](https://github.com/spring-projects/spring-data-bom/releases)
- [Commits](https://github.com/spring-projects/spring-data-bom/compare/2024.1.3...2024.1.4)

---
updated-dependencies:
- dependency-name: org.springframework.data:spring-data-bom
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
Bump org-eclipse-jetty from 11.0.24 to 11.0.25

Bumps `org-eclipse-jetty` from 11.0.24 to 11.0.25.

Updates `org.eclipse.jetty:jetty-server` from 11.0.24 to 11.0.25

Updates `org.eclipse.jetty:jetty-servlet` from 11.0.24 to 11.0.25

---
updated-dependencies:
- dependency-name: org.eclipse.jetty:jetty-server
  dependency-type: direct:production
  update-type: version-update:semver-patch
- dependency-name: org.eclipse.jetty:jetty-servlet
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

jzheaux committed:
Merge branch '6.4.x'

jzheaux committed:
Merge branch '6.4.x'
